tillerinobot/src/main/java/tillerino/tillerinobot/lang/TsundereGerman.java

```diff
@@ -0,0 +1,392 @@
+package tillerino.tillerinobot.lang;
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.NoResponse;
+import tillerino.tillerinobot.CommandHandler.Response;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+import tillerino.tillerinobot.diff.PercentageEstimates;
+
+public class TsundereGerman extends TsundereBase {
+	
+	//Random object, used in StringShuffler
+	static final Random rnd = new Random();
+	//Recent counters, reset if inactive for a while
+	int recentRecommendations = 0;
+	int recentHugs = 0;
+	
+	StringShuffler welcomeUserShortShuffler = new StringShuffler(rnd);
+	StringShuffler welcomeUserShuffler = new StringShuffler(rnd);
+	StringShuffler welcomeUserLongShuffler = new StringShuffler(rnd);
+	
+	@Override
+	public Response welcomeUser(OsuApiUser apiUser, long inactiveTime) {
+		String username = apiUser.getUserName();
+		String greeting = "";
+		//Greetings for <4 minutes, normal, and >4 days
+		if (inactiveTime < 4 * 60 * 1000) {
+			greeting = welcomeUserShortShuffler.get(
+				"Was soll das hier sein, das Kuckuckspiel?",
+				"Das waren nicht mal fünf Minuten...",
+				"Heute hat " + username + "an seinem Verschwindungszauber geübt."
```

Comment:
Agreed, but i couldn't think of a better one. I actually also think that Kuckuckspiel isnt quite as popular as a term for the game as peekaboo is in english